modules/covid_vaccine/app/jobs/covid_vaccine/confirmation_email_job.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+require 'va_notify/service'
+
+module CovidVaccine
+  class ConfirmationEmailJob
+    include Sidekiq::Worker
+    include SentryLogging
+    sidekiq_options expires_in: 1.day, retry: 2
+
+    STATSD_ERROR_NAME = 'worker.covid_vaccine_confirmation_email.error'
+    STATSD_SUCCESS_NAME = 'worker.covid_vaccine_confirmation_email.success'
+
+    def perform(email, date, sid)
+      submission = CovidVaccine::V0::RegistrationSubmission.find_by(sid: sid)
+      if submission.nil?
+        log_message_to_sentry('No SID found!', :warn, { email: email, sid: sid })
+        return
+      end
+      return if submission.email_confirmation_id.present?
+
+      if Flipper.enabled?(:vanotify_service_enhancement)
+        notify_client ||= VaNotify::Service.new(Settings.vanotify.services.va_gov.api_key)
+        template_id ||= Settings.vanotify.services.va_gov.template_id.covid_vaccine_registration
+      else
```

Comment:
Remove this else block - by the time we merge this we can just make sure we are putting the template ID in under the new preferred stanza in settings.yml.